runtime/tests/jni/module.xml

```diff
@@ -378,7 +378,7 @@
 			<include path="j9oti"/>
 		</includes>
 		<makefilestubs>
-			<makefilestub data="UMA_TREAT_WARNINGS_AS_ERRORS=1"/>
+			<makefilestub data="UMA_TREAT_WARNINGS_AS_ERRORS=0"/>
```

Comment:
Now I understand why this was added, I can say we don't want to deliver even this. Either fix the code in jnitest.c to remove the warning, or leave it for somebody else to fix.